xdsl/parser.py

```diff
@@ -304,19 +306,23 @@ def backtracking(self, region_name: str | None = None):
         with some meta information in the history attribute.
 
         The backtracker accepts the following exceptions:
-         - ParseError: signifies that the region could not be parsed because of (unexpected) syntax errors
-         - AssertionError: this error should probably be phased out in favour of the two above
+         - ParseError: signifies that the region could not be parsed because of
+         (unexpected) syntax errors
```

Comment:
Should we indent this line to make it more "readable" (not sure)

Reply:
yes